src/arcsf/data/data_utils.py

```diff
@@ -0,0 +1,244 @@
+import itertools
+
+import numpy as np
+from datasets import Dataset, load_dataset
+
+
+def get_indices_structured(
+    forgotten_author_numbers: list,
+    q_remove: int = 4,
+    n_authors: int = 200,
+    q_per_author: int = 20,
+):
```

Comment:
Still thinking about this but at least some of these values are TOFU constants and should be defined somewhere, e.g. `n_authors` and `q_per_author` I think will not change whenever using this with the original TOFU dataset so could be defined as `TOFU_N_AUTHORS`  and `TOFU_Q_PER_AUTHOR` and used throughout.

Reply:
Agreed re extraction to constants - happy for these to be used in main `load_tofu` fn but suggest leaving rest w/out defaults in case we use them when extending the data in #2